modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/js/liferay.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+// @ts-nocheck
```

Comment:
We should remove this before committing

Reply:
I would if I could figure out how to address the issue. Please pull this change and see what's wrong because I cannot figure it out. thx.